.pmd/ruleset.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<ruleset name="DOD"
+         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
+    <exclude-pattern>.*/compat/maven-compat/.*</exclude-pattern>
+    <rule ref="category/java/bestpractices.xml/MissingOverride" />
+    <!-- <rule ref="category/java/bestpractices.xml/UnusedAssignment" /> -->
+    <!-- <rule ref="category/java/bestpractices.xml/UnusedFormalParameter" /> -->
```

Comment:
don't comment code; just remove it

Reply:
yes will remove as takes some time and wanted to give ideas upfront. 

some of them already covered by check so we will find the blindspots.